<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>copyModuleDialog</class>
 <widget class="QDialog" name="copyModuleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>copyModuleDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="3,3,1,2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(210, 210, 210);</string>
       </property>
       <property name="text">
        <string>原命名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="oldNamelineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(210, 210, 210);</string>
       </property>
       <property name="text">
        <string>新命名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="newNamelineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>11</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="8,3,1,3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="SaveBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{color: rgb(255, 255, 255);border-top-color: rgb(180, 180, 180);
border-right-color: transparent;
border-left-color: transparent;
border-bottom-color:  transparent;}
QPushButton{background-color: rgb(87,94,136);color:white;}
QPushButton:hover{background-color: rgb(117,149,209);color:white;}
QPushButton:pressed{background-color: rgb(71, 175, 255);color:white;}</string>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="QuitBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{color: rgb(255, 255, 255);border-top-color: rgb(180, 180, 180);
border-right-color: transparent;
border-left-color: transparent;
border-bottom-color:  transparent;}
QPushButton{background-color: rgb(87,94,136);color:white;}
QPushButton:hover{background-color: rgb(117,149,209);color:white;}
QPushButton:pressed{background-color: rgb(71, 175, 255);color:white;}</string>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
